src/main/java/org/uwdigi/rag/service/SqlDatabaseContentRetriever.java

```diff
@@ -135,24 +137,33 @@ public SqlDatabaseContentRetriever(
       DataSource dataSource,
       String sqlDialect,
       String databaseStructure,
+      String sqlPromptTemplate,
       PromptTemplate promptTemplate,
```

Comment:
Hi @gitcliff can you investigate how to use this instead of introducing a new variable?